### PR TITLE
Added Signup and Login

### PR DESCRIPTION
**Pull request by MatthewLacerda2**
Company class now inherits from "Microsoft.AspNetCore.Identity.EntityFrameworkCore.IdentityUser", as account classes do in .NET Core
Heads up, .NET saves IdentityUsers in the dbo.AspNetUser table

Login is working and has a cookie

### CODE REVIEW

**Comment by pgbezerra** [comment]
This PR cannot be merged until the renaming is finished.

So, it should be a draft.

**Review by pgbezerra — COMMENTED** [review]
Please, rebase this code.
Then we can review it.

File: webserver/Models/Companies/Company.cs [hunk 1]
```diff
@@ -1,15 +1,10 @@
-﻿using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
-using Microsoft.AspNetCore.Identity;
+﻿using Microsoft.AspNetCore.Identity;
+using System.ComponentModel.DataAnnotations;
 
 namespace webserver.Models;
 
 public class Company : IdentityUser {
```
Comment (pgbezerra) [inline webserver/Models/Companies/Company.cs:6]: The company is not a user.
A user represents a single entity that can log in in a system.

A company is a generic entity, which can be represented by a user.

File: webserver/Models/Companies/Company.cs [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
 
-    [StringLength(20, MinimumLength = 3)]
-    [Required]
-    public string Name { get; set; } = string.Empty;
-
```
Comment (pgbezerra) [inline webserver/Models/Companies/Company.cs:12]: A company has a name.

File: webserver/Pages/Account/Login.cshtml [hunk 3]
```diff
@@ -34,4 +34,12 @@
...
     </form>
+</div>
+
+<div class="form-group row">
+    <div class="col-2">
+        <a href="\Account\Register" class="btn btn-primary">Register</a>
+    </div>
+    <div class="col-5">
+    </div>
 </div>
```
Comment (pgbezerra) [inline webserver/Pages/Account/Login.cshtml:45]: We don't want a register buttom.
This is a feature that we should do manually by the admin panel.

File: webserver/Models/Companies/Company.cs (hunk 2 above)
Comment (MatthewLacerda2) [reply webserver/Models/Companies/Company.cs:12]: Inherits from IdentityUser class

**Pull request closed without merging**